portfolio.js

```diff
@@ -0,0 +1,18 @@
+// const togglerButton = document.querySelector('#menuIcon');
+// const displayElement = document.querySelector('.desktop-list');
+// const exitIcon = document.querySelector('#exit-icon');
+// const body = document.querySelector('body')
+
+// togglerButton.addEventListener('click', () => {
+//   displayElement.style.display = 'block';
+//   togglerButton.style.display = 'none'
+//   console.log('#exitIcon')
+//   body.classList.toggle('open');
+
+// });
```

Comment:
- [ ] Kindly remove the commented-out code from this file. Removing such codes makes your work look clean. 👍